### ISSUE TITLE
Add support for xlsx and pdf downloads

### ISSUE THREAD

**Issue by crazycapivara** [eodaGmbH/py-tabulator#16]
See [downloads](https://tabulator.info/examples/5.5#download)

- [x] xlsx
- [ ] pdf

**Comment by j0bekt01** [comment]
you can do xlsx like this:
` ui.tags.script(type="text/javascript", src="https://oss.sheetjs.com/sheetjs/xlsx.full.min.js")`

`async with TabulatorContext("tabulator") as table:
          table.trigger_download("xlsx", "data.xlsx", {"sheetName":"sheetName"})`

**Comment by crazycapivara** [comment]
@j0bekt01 Yes, this is already implemented with [use_sheetjs](https://github.com/eodaGmbH/py-tabulator/blob/main/pytabulator/ui.py) as shown [in this example](https://eodagmbh.github.io/py-tabulator/examples/exports/).

Issue is still open because of the pdf support. Maybe I will split the issues.
